test/index.test.js

```diff
@@ -167,30 +134,30 @@ describe('index', () => {
         };
 
         beforeEach(() => {
-            breakRunMock.runCommand.yieldsAsync(null, fakeStopResponse, fakeStopResponse.body);
+            requestMock.yieldsAsync(null, fakeStopResponse, fakeStopResponse.body);
         });
 
         it('calls breaker with correct config', () => (
             executor.stop({
                 buildId: testBuildId
             }).then(() => {
-                assert.calledOnce(breakRunMock.runCommand);
-                assert.calledWith(breakRunMock.runCommand, deleteConfig);
+                assert.calledWith(requestMock, deleteConfig);
+                assert.calledOnce(requestMock);
             })
         ));
 
         it('returns error when breaker does', () => {
             const error = new Error('error');
 
-            breakRunMock.runCommand.yieldsAsync(error);
+            requestMock.yieldsAsync(error);
```

Comment:
Shouldn't we mock circuit breaker instead: `breakRunMock.runCommand.rejects(error)`